GPU/GPUTracking/Merger/macros/checkPropagation.C

```diff
@@ -1,11 +1,15 @@
-/*
- #include "Riostream.h"
- #include "TFile.h"
- #include "TMath.h"
- #include "TCanvas.h"
- #include "TH1F.h"
- #include "AliHLTTPCGeometry.h"
- */
+#if !defined(__CLING__) || defined(__ROOTCLING__)
```

Comment:
I thought the license statement is required (and checked by CI) also for macros

Reply:
The codechecker is not checking the macros.
@sgorbuno : I see that a lot of your macros have missing copyright statements. Could you add them?
Thanks
David